### ISSUE TITLE
Error in randomness

### ISSUE THREAD

**Issue by Elstuhn** [Elstuhn/Quantum-RNG#1]
To test the randomness, the distribution graph has to be uniform (we use KS test as well). 

**The problem**
Although the distribution is random, there is always a bias towards generating 0

**Recreating**
- Run test.py
- You will get a graph showing you a uniform distribution but then a huge spike at 0

**Task**
- Fix the program's bias towards 0
- In the description of your pull request describe why there is a bias towards 0 and how you fixed it (include technical details)

**Comment by avi-dey** [comment]
![test_result](https://github.com/Elstuhn/Quantum-RNG/assets/97702087/3ed08e9d-170c-401b-82a8-4c30efe59b28)
I have recreated this issue. I want to give it a try😀

**Comment by Elstuhn** [comment]
Yes, you've definitely recreated the issue. Go ahead and give it a try and
best of luck to you!

**Comment by avi-dey** [comment]
![test_result](https://github.com/Elstuhn/Quantum-RNG/assets/97702087/89162120-2fdd-4e55-bbd4-655c2fcd0f69)
Sorry, I deleted my comment because the image was too big 😂
yes, I would like to work on this. Can you assign me this?

**Comment by Elstuhn** [comment]
@avi-dey Sure, go ahead, I'll assign you the issue. Best of luck!
